hugr-core/src/hugr/rewrite/simple_replace.rs

```diff
@@ -793,7 +793,7 @@ pub(in crate::hugr::rewrite) mod test {
         };
         rewrite.apply(&mut hugr).unwrap_or_else(|e| panic!("{e}"));
 
-        assert_eq!(hugr.update_validate(&PRELUDE_REGISTRY), Ok(()));
+        assert_eq!(hugr.update_validate(&test_quantum_extension::REG), Ok(()));
```

Comment:
Whereas this might or might not be related, I'm really not sure. If it is required, sorry for noise!

Reply:
Tests fail without this. Because using only `Prelude` is actually invalid... (same with all the other test fixes).